releasenotes/notes/0.19/add-pulse-gate-pass-dc347177ed541bcc.yaml

```diff
@@ -0,0 +1,100 @@
+---
+features:
+  - |
+    A new transpiler pass group ``qiskit.transpiler.passes.calibration`` has
+    been added.  This group consists of
+
+    - :class:`~qiskit.transpiler.passes.RZXCalibrationBuilder`
+    - :class:`~qiskit.transpiler.passes.RZXCalibrationBuilderNoEcho`
+    - :func:`~qiskit.transpiler.passes.calibration.rzx_templates.rzx_templates`.
+    - :class:`~qiskit.transpiler.passes.PulseGates`
+
+    :obj:`.RZXCalibrationBuilder`, :obj:`.RZXCalibrationBuilderNoEcho`, and
+    ``rzx_templates`` were moved from ``qiskit.transpiler.passes.scheduling``,
+    whereas :obj:`.PulseGates` is the new pass that automatically extracts
+    user-provided calibrations from the instruction schedule map and attaches
+    the gate schedule to the given (transpiled) quantum circuit as a pulse gate.
+    Any transpiler pass that interacts with the pulse gate feature will be
+    grouped to ``qiskit.transpiler.passes.calibration``.
```

Comment:
I'm not sure about a lot of this note - the grouping of passes into `scheduling` or `calibration` is pretty meaningless from an end-user perspective if they're importing the way we suggest they do, with
```python
from qiskit.transpiler.passes import ...
```
because the group doesn't appear.

Reply:
Yeah we can just remove this piece as not user facing I think. We have the deprecation about the import path change below if people are using the full path for import so I think we don't really need to mention this as a new feature.

Reply:
Removed in a40c87764.